app/helpers/activity_helper.rb

```diff
@@ -30,14 +30,4 @@ def user_can_add_transaction?(user, activity)
     return true if user.delivery_partner? && activity.project?
     user.administrator? || user.fund_manager?
   end
-
```

Comment:
I think we could squash the earlier commit (f3d5c89a0f2df1ae1da2f326e9643c04d3fc829b) in which these methods were added and remove the need for a new commit here

Reply:
I tried so many times, I'm normally good with git hygiene but these two methods keep coming back! I'll keep trying